### ISSUE TITLE
panos_virtual_router (Resource): At most 1 occurrence is allowed for devices/entry"

### ISSUE THREAD

**Issue by Danmaarjustin** [PaloAltoNetworks/terraform-provider-panos#449]
## Steps to reproduce

1.
2.
3.
4.

## Your Environment

- Version used:
- Environment name and version (e.g. Chrome 59, node.js 5.4, python 3.7.3):
- Operating System and version (desktop or mobile):
- Link to your project:

**Comment by kklimonda-cl** [comment]
Could you try applying the resource with changes I've outlined below instead? 
I've explicitly set location.ngfw.ngfw_device to `localhost.localdomain` (which should be default and is the only value that makes sense there from what I understand).
The interface tunnel.5 must also be imported into virtual_router interfaces list so it can be used in the routing_table.

The modified resource:

```resource "panos_virtual_router" "example" {
  name = "default"

  location = {
    ngfw = {
      ngfw_device = "localhost.localdomain"
    }
  }

  interfaces = [
    "tunnel.5"
  ]

  routing_table = {
    ip = {
      static_routes = [
        {
          name        = "static-route-1"
          destination = "192.168.33.0/24"
          interface   = "tunnel.5"
        }
      ]
    }
  }
}```

**Comment by Justin-Schoenaker** [comment]
Yes this did the job, thnx a lot, also for te quick reply!! I tried this with no value but dint work.
Couldn't find it in the example or documentation, so was a bit lost. But this makes sense.

**Comment by rjaraw** [comment]
I'm also having a challenge adding new routes and interfaces to an existing virtual router. How do we import this resource? I looked online for samples but they don't seem to work to panos provider. It thinks I'm creating a new virtual router.

```
 Error: Error in create

  with panos_virtual_router.default,
  on tunnel.tf line 41, in resource "panos_virtual_router" "default":
  41: resource "panos_virtual_router" "default" {

entry from the plan already exists on the server
```
